erpnext/accounts/general_ledger.py

```diff
@@ -23,6 +25,28 @@ def make_gl_entries(gl_map, cancel=False, adv_adj=False, merge_entries=True, upd
 		else:
 			delete_gl_entries(gl_map, adv_adj=adv_adj, update_outstanding=update_outstanding)
 
+def validate_accounting_period(gl_map):
+	accounting_periods = frappe.db.sql(""" SELECT
+			ap.name as name
+		FROM
+			`tabAccounting Period` ap, `tabClosed Document` cd
+		WHERE
+			ap.name = cd.parent
+			AND ap.company = %(company)s
+			AND ap.status = "Open"
```

Comment:
This parent status is confusing, I think "Closed" checkbox in child is sufficient. Why not removing parent status field altogether?

Reply:
Added changes